src/common/tracer.h

```diff
@@ -16,6 +16,11 @@ using jspan_attribute = opentelemetry::common::AttributeValue;
 
 namespace tracing {
 
+static constexpr int TraceIdkSize = 16;
+static constexpr int SpanIdkSize = 8;
+static_assert(TraceIdkSize == opentelemetry::trace::TraceId::kSize);
```

Comment:
Why not just

```
static constexpr int TraceIdkSize = opentelemetry::trace::TraceId::kSize;
static constexpr int SpanIdkSize = opentelemetry::trace::SpanId::kSize;
```

Reply:
We do want to make sure we don't see those members change size since they're going over the wire.